### PR TITLE
Contract integration 3

### PR DESCRIPTION
**Pull request by adibas03**

### CODE REVIEW

File: src/components/pages/home/tab/profile/index.tsx [hunk 1]
```diff
@@ -14,41 +14,60 @@ import WhyMe from "./whyMe";
...
-      );
-    this.setState({repayments});
-    } catch (err) {
-      console.log(err)
-    }
+      this.setState({repayments});
+      } catch (err) {
+        console.log(err)
+      }
+    }, 3000);
```
Comment (djuanit0x) [inline src/components/pages/home/tab/profile/index.tsx]: Why do you use setTimeOut? @adibas03

File: src/utils/web3Utils.js [hunk 2]
```diff
@@ -1,5 +1,11 @@
+import Web3 from 'web3';
 import getWeb3 from './getWeb3';
 
+const getAccounts = async (web3) => {
+    web3 = web3 || await getWeb3();
+    return await web3.eth.getAccounts();
+}
```
Comment (djuanit0x) [inline src/utils/web3Utils.js]: I use a similar util function [here ](https://github.com/enabledao/enable-ui/blob/contract-integration-2/src/utils/web3Utils.js#L23)

File: src/components/pages/home/tab/profile/index.tsx (hunk 1 above)
Comment (adibas03) [reply src/components/pages/home/tab/profile/index.tsx]: I noticed Metamask has an issue with collusion of RPC calls, which was causing the Node Errors we were getting intermittently. The timeout stops that temporarily, while we think out a permanent fix

File: src/utils/web3Utils.js (hunk 2 above)
Comment (adibas03) [reply src/utils/web3Utils.js]: yea,
saw that.
Yours returns the expected default address, as against all addresses

**Pull request merged**